### PR TITLE
Docker to run all services

### LINKED ISSUE #65: Run celery and flask with single command
Find a way to run celery and flask app together. Best possible way is to use docker.

### PR DESCRIPTION
**Pull request by karelumair**
Added docker support to run all services at a time.

closes #65

### CODE REVIEW

**Comment by pratikGhodke1** [comment]
Update the commit to use `feat`

**Comment by karelumair** [comment]
> Update the commit to use `feat`

Its already **feat**

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+__pycache__
+.vscode/
+.venv/
+.env
+.pytest_cache/
+logs.log
+geckodriver.log
+Driver/
```
Comment (pratikGhodke1) [inline .dockerignore]: Check if these can be removed

**Review by pratikGhodke1 — CHANGES_REQUESTED** [review]
Please check the comment

**Comment by karelumair** [comment]
> Please check the comment

Done

**Review by pratikGhodke1 — APPROVED** [review]

**Pull request merged**